_projects/git_task_list.md

```diff
@@ -0,0 +1,70 @@
+---
+collaborating_projects:
+  - gsoc-prep-tasks
+desc: "List pull request/merge request (in GitLab) that have no reviewer yet. Provide an easy way for newcomer / developer to do contribution to a project by giving review to other people pull requests."
+difficulty: medium
+initiatives:
+  - GSoC
+issues:
+  - https://github.com/coala/projects/issues/480
+  - https://github.com/coala/coala/issues/3615
+  - https://github.com/coala/coala/issues/4080
+markdown: git_task_list.md
+mentors:
+  - sims1253
+  - yukiisbored
+  - yashovardhanagrawal
+name: "Git Task List"
+requirements:
+  - "The applicant should have at least one patch merged to any of the coala repositories."
+  - "Familiarity with Git, GitHub API Gerrit API, Phabricator API, and GitLab API"
+  - "Familiarity with EmberJS, Javascript, and GitHub Pages"
+status: in_progress
+tags:
+  - community
+  - web
+---
+The basic idea is an extended version of [GSOC Task List](http://summerofcode.github.io/gsoc-prep-tasks)
+which was initialy intended for GSOC Applicants who don't have or only have a
+litle experience in open-source. Hence, [that](http://github.com/summerofcode/gsoc-prep-tasks)
+project will be the starting point of this project.
+
+This project will help sharing the tasks of reviewing a pull request on a
```

Comment:
Before discussing this project, describe what gsoc-prep-tasks tool is in more detail, technically and conceptually.

Also, your project description here should summarise the objectives of the milestone features below, which need to be roughly the same as the milestone features in your GSoC proposal.

gsoc-prep-tasks does not currently help very much in the process of reviewing a pull request, and the features you planned, like 'Add GitLab/Gerrit/Phabricator' are not going to help in the process of reviewing a pull request.

If you intended to "help sharing the tasks of reviewing a pull request", you have to explain how you are going to improve gsoc-prep-tasks to do that!

If someone new tries to read this project page, they need to be able to understand what you are going to do for the next three months.

A fundamental question is whether 'gsoc-prep-tasks' is going to be 'only issues' (this is a current limitation), or also will include 'pull requests' (iirc there is a issue about that in your repo).

If it is going to include pull requests, that is something you need to add as one of your first milestone tasks section of this .md file.